demo/package.json

```diff
@@ -20,12 +20,17 @@
   },
   "devDependencies": {
     "@citizensadvice/design-system": "file:../",
-    "@testing-library/cypress": "^10.0.2",
+    "@testing-library/cypress": "^10.0.3",
     "axe-core": "^4.10.2",
     "backstopjs": "^6.3.25",
-    "cypress": "^13.17.0",
+    "cypress": "^14.0.0",
     "cypress-axe": "^1.5.0",
     "cypress-plugin-tab": "^1.0.5",
     "start-server-and-test": "^2.0.9"
+  },
+  "overrides": {
+    "cypress-axe": {
+      "cypress": "$cypress"
+    }
```

Comment:
Using an override here feels fine to me here as this library is so small (so both is very easy to inline if we need to but also means it might not get a new release for a little bit as it's not regularly updated)